public/components/content-list-drawer/content-list-drawer.html

```diff
@@ -349,6 +338,38 @@
 
             </div>
 
+            <div class="drawer__section" ng-class="{'drawer__section--closed' : openSection !== 'usages'}" ng-if="contentItem.contentType === 'media'">
+                <div class="drawer__header" ng-class="{'drawer__header--closed' : openSection !== 'usages'}" ng-click="toggleSection('usages')">
+                    <i class="drawer__icon" wf-icon="usages" />
+                    Usages
+                </div>
+
+                <div class="drawer__inner" ng-class="{'drawer__inner--closed' : openSection !== 'usages'}" >
+
+                    <ul class="drawer__column drawer__column--wide">
+                        <li class="drawer__item">
+                            <ul class="drawer__item-content usages-list" ng-if="capiData.usages.length">
+
+                                <li class="usage" ng-class="{'usage--highlight' : usage.type === 'video'}" ng-repeat="usage in capiData.usages">
+                                    <p class="usage__title">{{ usage.webTitle || '-' }}</p>
+                                    <div class="usage__details">
+                                        <span class="usage__date">{{ 'Created: ' + usage.friendlyDate }}</span>
+                                        <a href="{{ usage.composerUrl }}" target="_blank"><i class="drawer__icon usage__icon" wf-icon="composer"></i></a>
+                                        <a href="{{ usage.viewerUrl }}" target="_blank"><i class="drawer__icon usage__icon" wf-icon="preview"></i></a>
```

Comment:
Maybe we could add a `title` attr to these anchors with some more info of where they go?